### PR TITLE
Add install command and manifest sub command

### PR DESCRIPTION
**Pull request by hanahmily**
Add new command to generate manifest would be applied to Kubernetes.

For more details, pls read REAME.md

Signed-off-by: Gao Hongtao <hanahmily@gmail.com>

### CODE REVIEW

**Comment by hanahmily** [comment]
Once https://github.com/apache/skywalking-swck/pull/19 is merged, this PR could be reviewed.

File: go.mod [hunk 1]
```diff
@@ -2,26 +2,23 @@ module github.com/apache/skywalking-cli
 
 go 1.13
 
+replace github.com/apache/skywalking-swck => github.com/hanahmily/skywalking-swck v0.0.0-20210106071233-5406527abfa3
```
Comment (wu-sheng) [inline go.mod:5]: What is this?

Comment (hanahmily) [reply go.mod:5]: To dependent my private dev branch temporarily which doesn't be merged, that's why I mark this PR as 'WIP'

File: Makefile [hunk 2]
```diff
@@ -83,7 +83,7 @@ $(PLATFORMS):
 
 .PHONY: lint
 lint: codegen tools
-	$(GO_LINT) run -v ./...
+	$(GO_LINT) run -v --timeout 5m ./...
```
Comment (kezhenxu94) [inline Makefile:86]: Just curious, under what kind of cases will the linter cost this much time?

File: README.md [hunk 3]
```diff
@@ -620,6 +636,95 @@ $ ./bin/swctl checkHealth --grpcTLS=true
...
+```
+
+Load overlay custom resource from stdin
+```shell
+cat ui-cr.yaml | ./bin/swctl install manifest oap -f=-
+```
+
+Apply directly to Kubernetes
+```shell
+./bin/swctl install manifest oap -f oap-cr.yaml | kubectl apply -f-
```
Comment (kezhenxu94) [inline README.md:660]: `-f-` need a space or `=` between `-f` and `-`?

File: internal/commands/install/manifest/manifest.go [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+	r := repo.NewRepo(repoName)
+	ff, err := r.GetFilesRecursive("templates")
+	if err != nil {
+		return fmt.Errorf("failed to load resource templates: %v", err)
+	}
+
+	for _, f := range ff {
+		manifests, err := r.ReadFile(f)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to read template ile content: %v", err)
```
Comment (kezhenxu94) [inline internal/commands/install/manifest/manifest.go:123]:
```suggestion
			fmt.Fprintf(os.Stderr, "failed to read template file content: %v", err)
```

Comment (kezhenxu94) [reply internal/commands/install/manifest/manifest.go:123]: And use `logger` to print the error messages?

File: Makefile (hunk 2 above)
Comment (hanahmily) [reply Makefile:86]: under GitHub action, the linter will use more than 1m which is the default value of golang-ci lint.

File: assets/cr/ui.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+apiVersion: operator.skywalking.apache.org/v1alpha1
+kind: UI
+metadata:
+  name: minimal
```
Comment (kezhenxu94) [inline assets/cr/ui.yaml:21]: the name `minimal` seems to be not related to the `UI`? Is it intentional?

File: README.md (hunk 3 above)
Comment (hanahmily) [reply README.md:660]: Sadly for the CLI lib that we're using, space or `=` seems not redundant. But it doesn't when I use cobra. I didn't dive into why this happened.

**Comment by kezhenxu94** [comment]
Others look good to me

File: internal/commands/install/manifest/manifest.go (hunk 4 above)
Comment (hanahmily) [reply internal/commands/install/manifest/manifest.go:123]: I tried `logger`, but it doesn't print messages to stderr. If they're printed into stdout, the manifest will be messed up. Do you have some idea how to redirect the error log into stderr?

File: assets/cr/ui.yaml (hunk 5 above)
Comment (hanahmily) [reply assets/cr/ui.yaml:21]: sorry, it should be `skywalking`. let me fix it.

**Review by kezhenxu94 — DISMISSED** [review]

**Review by kezhenxu94 — APPROVED** [review]
